### ISSUE TITLE
Clipping or normalizing gradients

### ISSUE THREAD

**Issue by FerranAlet** [facebookresearch/higher#50]
Is there a way to add gradient clipping or gradient normalization on the inner loop optimization? Thanks!

**Comment by jkkronk** [comment]
Hi, If I'm not wrong you could use grad_callback as argument when step for this.

> grad_callback – (optional) a single argument function which will be applied to a list of gradients of parameters, which respects the order specified by reference_params. This can be used to apply a function, such as gradient clipping, to all (or a subset) of these gradients every time the step function is called. If this keyword argument is provided when calling the step method, its value will override the default specified here.

**Comment by egrefen** [comment]
@FerranAlet: @jkkronk is right that the `grad_callback` argument is supposed to provide this. Does it solve your problem?

**Comment by egrefen** [comment]
I will close this issue assuming it is resolved if I don't hear back in one month.

**Comment by FerranAlet** [comment]
Sorry for not replying back in the day; I believe it did solve the issue, thanks! @jkkronk @egrefen
